src/explore-education-statistics-admin/src/pages/release/data/components/__tests__/ApiDataSetPreviewTokenCreateForm.test.tsx

```diff
@@ -0,0 +1,123 @@
+import ApiDataSetPreviewTokenCreateForm from '@admin/pages/release/data/components/ApiDataSetPreviewTokenCreateForm';
+import render from '@common-test/render';
+import { screen, waitFor } from '@testing-library/react';
+import noop from 'lodash/noop';
+
+describe('ApiDataSetPreviewTokenCreateForm', () => {
+  test('renders the form correctly', () => {
+    render(
+      <ApiDataSetPreviewTokenCreateForm onCancel={noop} onSubmit={noop} />,
+    );
+
+    expect(
+      screen.getByLabelText('Name this token instance'),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByLabelText(
+        'I agree to only share this token with individuals that have been granted production access to the unpublished data',
```

Comment:
Minor, but could avoid having to copy this entire thing in verbatim by using a regex e.g.

```ts
screen.getByLabelText(/I agree/);
```
This would be a little more robust to wording changes so we don't need to update the tests when it changes.

Similar could be done later in this test suite as well.